clang/test/Misc/diag-template-diffing-cxx11.cpp

```diff
@@ -80,7 +80,7 @@ void test2() {
 // CHECK-ELIDE-TREE: candidate function not viable: no known conversion from argument type to parameter type for 1st argument
 // CHECK-ELIDE-TREE:   I2<
 // CHECK-ELIDE-TREE:     [double != int],
-// CHECK-ELIDE-TREE:     [...], 
```

Comment:
There are a lot of merely formatting changes here, it is already a large PR and it feels distracting, I would have preferred this was a separate NFC commit to fix the formatting.